Dockerfile

```diff
@@ -8,6 +8,8 @@ EXPOSE 5000
 
 ENV DATABASE_URL="jdbc:sqlite:./vanilla_db"
 
-COPY vanilla_db vanilla_db
+# Copy the default, or fresh, Database over and use that in the new container
+# By Copying it to vanilla_db it now can be used by our URI handlers
+COPY vanilla_default vanilla_db
```

Comment:
is this the correct "direction" I never remember

Reply:
should be in Windows.  Don't know about you Apple people...

If you're brave, you can use /Y and suppress warnings about overwriting a file.

Reply:
according to https://docs.docker.com/engine/reference/builder/, it is COPY <source> <destination>, this is correct

Reply:
This is what I thought, but I really was not sure when I was doing it. It seemed like I was just guessing even after reading the documentation, so thank you for this input!